src/services/db/GitFileCommitService.ts

```diff
@@ -161,50 +167,41 @@ export default class GitFileCommitService {
     logger.info(
       `Deleting directory in local Git file system for repo: ${sessionData.siteName}, directory name: ${directoryName}`
     )
-    const deletePromises = [
-      this.gitFileSystemService.delete(
-        sessionData.siteName,
-        directoryName,
-        "",
-        sessionData.isomerUserId,
-        true,
-        STAGING_BRANCH
-      ),
-    ]
+    const stagingDeleteResult = await this.gitFileSystemService.delete(
+      sessionData.siteName,
+      directoryName,
+      "",
+      sessionData.isomerUserId,
+      true,
+      defaultBranch
```

Comment:
nit: feels like we can just use `STAGING_BRANCH` here directly. Same for the subsequent functions.

Reply:
HMM ok reason why I made this compulsory was cuz i forgot to reset the default in a control flow. I rather code be verbose but caller is explicit about this rather than simplistic and errors are hidden here. lmk if you feel strongly about this uh